change/@internal-calling-component-bindings-5ea8a8ee-352d-4df0-b056-bb3c982e0fba.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add missing foreveryone logic for leaveCall",
```

Comment:
should ideally be 'none' since it's a bug fix.
Starting the comment with the text "Bugfix" makes it easier to tweak the changeling during release.

Reply:
bug fixes should be of type `patch` (`none` excludes them from changelog generation)

Reply:
Ah yes. My bad.
This is correct.